common/browser.go

```diff
@@ -242,6 +242,11 @@ func (b *Browser) onAttachedToTarget(ev *target.EventAttachedToTarget) {
 	}
 
 	session := b.conn.getSession(ev.SessionID)
+	if session == nil {
+		b.logger.Debugf("Browser:onAttachedToTarget seesion got closed before attachToTarget was handled",
+			"sid:%v tid:%v", ev.SessionID, evti.TargetID)
```

Comment:
IMO, this should produce a warning for us to catch it later on when it happens.

```suggestion
		b.logger.Warnf("Browser:onAttachedToTarget",
			"session closed before attachToTarget is handled. sid:%v tid:%v",
			ev.SessionID, evti.TargetID)
```